dotcom-rendering/src/lib/byline.ts

```diff
@@ -29,9 +31,11 @@ export const getSoleContributor = (
 	 */
 	const [soleContributor, ...otherContributors] = tags.filter(isContributor);
 
+	if (!byline) return undefined;
 	if (otherContributors.length !== 0) return undefined;
+	if (byline.includes(' and ')) return undefined;
```

Comment:
This is potentially genius. If we haven't already, it might be worth passing it by CP to both get their input and also to make them aware this is how things will work

Reply:
> “potentially genius”

👨‍🔬 

> it might be worth passing it by CP to both get their input and also to make them aware this is how things will work

Having a chat with some of them tomorrow.

Reply:
Gabriel Smith and Matthew Holmes were happy with this implementation.